kod/util/rntmaint.kod

```diff
@@ -169,7 +175,26 @@ messages:
             #parm1=initCost, #parm2=perDayCost, #parm3=keyCopyCost);
 
          return;
-      }    
+      }
+      
+      % Command: Change Lock
+      if StringContain(speech, RoomMaintenance_command_change_lock)
+         OR StringContain(speech, "change lock")
```

Comment:
Isn't this checking for the same thing twice?

Reply:
It is, but the other commands were like that as well so I just followed what I saw to be the pattern.  It seems kind of pointless to include them twice though, so if you want, I'll remove the extra check from this and the other commands as well.

Reply:
Yes, please fix this everywhere you see it (after testing to make sure it still works, of course).  It's worse than pointless: if you ever decide to rename "change lock" to something else, you go and change the resource, but "change lock" still works?!?!

Reply:
Done, and gladly so.